### PR TITLE
Update package.json to include a `unpkg` field

### PR DESCRIPTION
**Pull request by kraenhansen**
## What, How & Why?

This adds a `unpkg` field to the package.json to simplify the URL users need to paste when consuming the IIFE bundle using unpkg.com.

## ☑️ ToDos
* [x] 📝 Changelog entry
* [x] 📝 `Compatibility` label is updated or copied from previous entry
* [ ] 🚦 Tests
* [x] 📱 Check the React Native/other sample apps work if necessary
* [x] 📝 Public documentation PR created or is not necessary
* [x] 💥 `Breaking` label has been applied or is not necessary

### CODE REVIEW

File: packages/realm-web/scripts/postversion.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ const changelogPath = path.resolve(__dirname, "../CHANGELOG.md");
 
 // Update the readme file to use the new version in the script-tag.
 const readmeContent = fs.readFileSync(readmePath, "utf8");
-const readmeContentReplaced = readmeContent.replace(/realm-web@[^/]+/, `realm-web@${version}`);
+const readmeContentReplaced = readmeContent.replace(/realm-web@[^"/]+/, `realm-web@${version}`);
```
Comment (tomduncalf) [inline packages/realm-web/scripts/postversion.ts:33]: Is this change deliberate?

**Review by tomduncalf — APPROVED** [review]

**Review by kneth — APPROVED** [review]
LGTM

File: packages/realm-web/scripts/postversion.ts (hunk 1 above)
Comment (kraenhansen) [reply packages/realm-web/scripts/postversion.ts:33]: Yeah. Now that the version is the last part of the URL it needed a " in the pattern.
